### PR TITLE
.NET Interactive: Enable_try_dotnet_to_batch_8d

### PR DESCRIPTION
**Pull request by v-mepa**
**Enable_try_dotnet_to_batch_8d**

URL | Comments
-- | --
https://docs.microsoft.com/en-us/dotnet/api/system.char.getunicodecategory | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.isseparator |  
https://docs.microsoft.com/en-us/dotnet/api/system.char.islower | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.ishighsurrogate | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.compareto | **Enable only for the second example. First sample   is culture dependent**.
https://docs.microsoft.com/en-us/dotnet/api/system.char.issurrogate | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.islowsurrogate | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.issurrogatepair | Duplicate samples for all overloads? Potentially   move it to the MemberGroup

### CODE REVIEW

**Comment by v-mepa** [comment]
fyi @mairaw , let me know how it looks.

File: xml/System/Char.xml [hunk 1]
```diff
@@ -1012,12 +1012,12 @@ When a managed <xref:System.Char> type, which is represented as a Unicode UTF-16
 
 
 
-## Examples
+ ## Examples
```
Comment (mairaw) [inline xml/System/Char.xml:1015]: This needs to be moved to the MemberGroup element

File: xml/System/Char.xml [hunk 2]
```diff
@@ -1956,13 +1938,13 @@ When a managed <xref:System.Char> type, which is represented as a Unicode UTF-16
  Valid lowercase letters are members of the following category in <xref:System.Globalization.UnicodeCategory>: `LowercaseLetter`.
 
 
-
+ 
 ## Examples
```
Comment (mairaw) [inline xml/System/Char.xml:1942]: this needs to be moved to the MemberGroup element

File: xml/System/Char.xml [hunk 3]
```diff
@@ -2112,9 +2085,9 @@ When a managed <xref:System.Char> type, which is represented as a Unicode UTF-16
...
```
Comment (mairaw) [inline xml/System/Char.xml:2085]: This needs to be moved to the MemberGroup element

File: xml/System/Char.xml [hunk 4]
```diff
@@ -2851,9 +2815,9 @@ When a managed <xref:System.Char> type, which is represented as a Unicode UTF-16
...
```
Comment (mairaw) [inline xml/System/Char.xml:2815]: This needs to be moved to the MemberGroup element

File: xml/System/Char.xml [hunk 5]
```diff
@@ -1465,13 +1456,13 @@ When a managed <xref:System.Char> type, which is represented as a Unicode UTF-16
  In addition to representing single characters using a 16-bit code point, UTF-16 encoding allows abstract characters to be represented using two 16-bit code points, which is known as a surrogate pair. The first element in this pair is the high surrogate. Its code point can range from U+D800 to U+DBFF. An individual surrogate has no interpretation of its own; it is meaningful only when used as part of a surrogate pair.
 
 
-
+ 
 ## Examples
```
Comment (mairaw) [inline xml/System/Char.xml:1460]: This needs to be moved to the MemberGroup element

File: xml/System/Char.xml [hunk 6]
```diff
@@ -3001,9 +2956,9 @@ When a managed <xref:System.Char> type, which is represented as a Unicode UTF-16
...
```
Comment (mairaw) [inline xml/System/Char.xml:2956]: This needs to be moved to the MemberGroup element

**Pull request merged**